kong/identifiers.go

```diff
@@ -0,0 +1,91 @@
+package kong
+
+import "fmt"
+
+// Identifier returns the endpoint key name or ID.
+func (s1 *Service) Identifier() string {
```

Comment:
Please call this function `FriendlyName` to better describe its purpose.

Reply:
There is already Identifier or Console which behave the same way in deck, are you sure about this name?

Reply:
It does, I'll take care of this this weekend